ground/src/main/java/com/google/android/ground/ui/map/gms/renderer/PolylineRenderer.kt

```diff
@@ -33,16 +38,26 @@ class PolylineRenderer(
 
   private val polylines: MutableMap<Feature, MutableList<Polyline>> = HashMap()
 
-  override fun addFeature(feature: Feature, geometry: Geometry) {
-    val points = geometry.vertices
+  override fun addFeature(feature: Feature) {
+    when (feature.geometry) {
+      is Point,
+      is MultiPolygon,
+      is Polygon ->
+        throw IllegalArgumentException("Can't render polyline for geometry: ${feature.geometry}")
```

Comment:
Perhaps these should be covered by `else`? I'd also update the error to something more descriptive, e.g.

`error("PolylineRendered expected LineString or LinearRing, but got ${feature.geometry::class.simpleName}")`

Reply:
Done